CMakeLists.txt

```diff
@@ -365,7 +365,7 @@ target_link_libraries(core_interface INTERFACE sanitize_interface)
 if(SANITIZERS)
   # First check if the compiler accepts flags. If an incompatible pair like
   # -fsanitize=address,thread is used here, this check will fail. This will also
-  # fail if a bad argument is passed, e.g. -fsanitize=undfeined
+  # fail if a bad argument is passed, e.g. -fsanitize=undefined
```

Comment:
It's not a typo, rather an example of literally _bad_ argument.

Compare: https://github.com/hebasto/bitcoin/blob/3c200d5d1a17636416c7b7382a2ae44d5fda59b3/configure.ac#L351-L353

Reply:
Heh, I need to take a break!